src/components/Modal/readme.md

```diff
@@ -708,3 +708,276 @@ class FormModal extends React.Component {
     </GlobalHeader>
 </div>
 ```
+
+##### make reservation modal
+
+```js
+import React from 'react';
+import { Modal, Button, Input, TimePicker, DatePicker, Textarea, Select } from 'react-rainbow-components';
+import { Field, reduxForm } from 'redux-form';
+import styled from 'styled-components';
+
+const Content = styled.div.attrs( props => ({
+    imgWidth: '328px'
+}))`
+    div:nth-child(2) img {
+        min-width: ${props => props.imgWidth};
+    }
+
+    @media (max-width: 768px) {
+        flex-direction: column-reverse;
+        
+        > div:nth-child(1) {
+            text-align: center;
+        }
+        
+        div:nth-child(2) img {
+            min-width: 0;
+            width: ${props => props.imgWidth};
+            max-width: 100%;
+        }
+    }
+`;
+
+const Title = styled.h2.attrs(props => {
+   return props.theme.rainbow.palette;
+})`
+    font-size: 36px;
+    font-weight: 300;
+    line-height: 'normal';
+    color: ${props => {
+        return (props.brand.main === '#80deea') ? '#fff' : '#061c3f';
```

Comment:
we already have a helper function that comes in theme palette  and this function computes the right text color based on background color passed
```color: ${props.getContrastText(props.background.secondary)};```